### ISSUE TITLE
Buttons aren't WCAG 2.1 AA compatible

### ISSUE THREAD

**Issue by robsonsobral** [browser-update/browser-update#417]
Hi!

The buttons doesn't have enough text contrast.

May I submit a pull request?

**Comment by josselex** [comment]
Is there a way to adress only users that need this accessability? Like some
css media query?

Am Mi., 10. Okt. 2018 um 14:14 Uhr schrieb Robson Sobral <
notifications@github.com>:

**Comment by robsonsobral** [comment]
HI, @josselex .

No, it doesn't. There's no way (yet) to know the user disabilities and target them.

Thank you.

**Comment by pa-jkeyser** [comment]
It is not possible to know who viewing a web site has low vision or is color blind to know who needs more color contrast to read some text. All text should have enough contrast to cover all of those will vision problems of some kind.

In addition to the color contrast accessibility issues, I also noticed that the "Update browser" button does not have a "role" attribute although the "Ignore" button does.
